data/issues.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import active_newcomers
+from data.models import Contributor
+from data.webservices import webservices_url
+from data.convert import convert
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    hoster = hoster.lower()
+    if hoster == 'github':
+        IMPORT_URL = webservices_url('issues/github/all')
+    elif hoster == 'gitlab':
```

Comment:
and if hoster isnt either of these ... ?
use:
```py
IMPORT_URL = webservices_url('issues/%s/all' % hoster)
```

then an unexpected value will cause a failure.